### DISCUSSION TITLE
Change 12/24hour time / clock format

### DISCUSSION THREAD

**Discussion by Paradoxum** [mathoudebine/turing-smart-screen-python#228] [Q&A]
Change 12/24hour time / clock format etc?

https://github.com/mathoudebine/turing-smart-screen-python/wiki/System-monitor-:-themes

Doesn't seem like it's possible but maybe i'm blind and missing an option. Anyone?

Thank you

**Comment by mathoudebine** [comment]
Hi! There is no settings for this now, the system monitor should use the locale you have set on your OS.
May I ask why you want to change it? Is it because the date/time does not match the one from your system? Or you just want a different format than your system?

**Comment by fireball2k2000** [comment]
On Linux, you can set the locale specific environments before calling the main.py:

`LC_TIME=de_DE ./main.py`

You can also modify the systemd .service file provided:

```
WorkingDirectory=/opt/turing-smart-screen-python/
ExecStart=python3 main.py
Environment=LC_TIME=de_DE
```


I'm also using that option because my general system locale is set to en_US and I like to have German time display (24h instead of AM/PM) for convenience ;)
